app/models/training/page.rb

```diff
@@ -5,6 +5,11 @@ def self.content_type_id
       'page'
     end
 
+    # @return [Boolean]
+    def updated_content?
```

Comment:
What improvement could you make so that `Question` can also use this?

Reply:
I have moved this to the Content model